src/editors/Flow/view/Components/FlowTopBar/FlowSearch.jsx

```diff
@@ -79,16 +80,15 @@ const FlowSearch = props => {
           className={classes.searchInputText}
           autoFocus
           InputProps={{
-            ...params.InputProps,
             startAdornment: (
-              <>
-                <InputAdornment position="start">
-                  <SearchIcon />
-                </InputAdornment>
-                {params.InputProps.startAdornment}
-              </>
+              <InputAdornment position="start">
```

Comment:
The dropdown with the existent matching nodes is not appearing

Reply:
Now it is fixed.